Source/Additions/Foundation/NSNumber+BFKit.h

```diff
@@ -42,15 +42,15 @@ extern "C" {
  *
  *  @return Returns the convertion result
  */
-CGFloat DegreesToRadians(CGFloat degrees);
+CGFloat BFDegreesToRadians(CGFloat degrees);
```

Comment:
Why do you need to change this function? This is a breaking change, if we will do it, we will need to create a 2.0 version, because of semantic versioning.
Same thing applies to `BFRadiansToDegrees`